ui/src/app/utils/strings.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * This method will Extract a String based on the given delimeter and its occurence
+ * @function
+ * @param string 'www.google.com'
+ * @param delimeter '.'
+ * @param occurence '1' //if you want last occurence then you have to give -1
+ *
+ * @return string "google.com"
+ *
+ */
+interface extractStringByDelimiterByPosArguments{
+  str:string;
+  delimiter?:string;
+  occurence?:number;
+}
+
+export const extractStringByDelimiterByPos = (args:extractStringByDelimiterByPosArguments) => {
+  let defaultValues={delimiter:".",occurence:-1}
+  args={...defaultValues,...args}
```

Comment:
this will overwrite the delimiter with default values, even if we send other value, can you plz verify?